### DISCUSSION TITLE
Qual melhor curso de Angular disponível atualmente?

### DISCUSSION THREAD

**Discussion by hiiago21** [frontendbr/forum#2039] [Perguntas e Respostas]
Estou querendo ampliar meu conhecimento em Angular e gostaria de saber na opinião de vocês qual o melhor curso do mercado para Angular desde o básico até o avançado.

**Comment by marcuspaulo** [comment]
Olá Hiiago21, 

Seguem algumas sugestões: 

Esse curso é frequentemente atualizado, Brasileiros conseguem um bom desconto no curso vitalício.
https://angular-university.io/

Apesar de ser um curso um pouco antigo, o professor passa umas dicas bem interessantes.
Construindo Aplicações Web Com o Novo Angular (4, 5 e 6)
https://www.udemy.com/course/angular-pt/

Curso Angular - Loiane Groner
https://loiane.training/curso/angular

Não tenho referências sobre esse curso.
https://ultimatecourses.com/learn/angular-pro


Bons estudos.

**Reply by hiiago21** [reply]
Baah, muito obrigado vou olhar sim esses conteúdos

**Comment by raloliver** [comment]
Aprendi muito de angular com bons professores, vou recomendar dois aqui:

1. **Flávio Almeida**: https://cursos.alura.com.br/user/flavio-almeida
2. **Leonardo  Leitão**: https://www.cod3r.com.br/courses/angular-9-essencial

Se você conseguir absorver conteúdo em inglês, tem esses dois:

1. **Maximilian Schwarzmüller**
2. **Deborah Kurata**

Além claro, dos que já foram citados pelo @marcuspaulo  ;)

**Reply by hiiago21** [reply]
muito obrigado, o da alura realmente é muito bom já estou fazendo o mesmo

**Comment by dziul** [comment]
Curto o ensinamento da Loiane Groner, além de Excelente conteúdo é _free_:

- https://loiane.training/curso/angular
- https://www.youtube.com/watch?v=tPOMG0D57S0&list=PLGxZ4Rq3BOBoSRcKWEdQACbUCNWLczg2G&ab_channel=LoianeGroner

**Reply by hiiago21** [reply]
muito obrigado, vou dar uma olhada no conteúdo dela!
